verify/verify.go

```diff
@@ -54,6 +54,7 @@ func (verifier *ReproducibleProvenanceVerifier) Verify(provenanceFilePath string
 		return fmt.Errorf("couldn't load BuildConfig from provenance: %v", err)
 	}
 
+	tempFolder := ""
```

Comment:
What's the plan for deduplicating this method vs the other above? Is there an issue for that already?